inspirehep/modules/records/serializers/__init__.py

```diff
@@ -87,11 +89,22 @@
     'application/json',
 )
 
+json_aggregations_ui_v1 = FacetsJSONUISerializer(
```

Comment:
Maybe better use 'json_literature_aggregations_ui'? Will we want to have one for each type of record?